sdk/keyvault/azure_security_keyvault_certificates/README.md

```diff
@@ -209,7 +209,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     )?;
 
     let certificate = client
-        .get_certificate("certificate-name", "certificate-version", None)
+        .get_certificate("certificate-name", None)
```

Comment:
Nit, I think the doc comment on line 195 needs to be updated to reflect the new public surface area (probably applies elsewhere).

Reply:
Ok, I have the fix for this one (the others in the certificates readme aren't a problem). But it'll need a re-approval.